batavia/modules/math.js

```diff
@@ -0,0 +1,201 @@
+
+batavia.modules.math = {
+    __doc__: "",
+    __file__: "math.js",
+    __name__: "math",
+    __package__: "",
+    e: new batavia.types.Float(Math.E),
+    nan: new batavia.types.Float(NaN),
+    pi: new batavia.types.Float(Math.PI),
+    inf: new batavia.types.Float(Infinity),
+
+    acos: function(x) {
+        return new batavia.types.Float(Math.acos(x.__float__().val));
+    },
+
+    // TODO: this isn't supported in PhantomJS
+    acosh: function(x) {
+        return new batavia.types.Float(Math.acosh(x.__float__().val));
+    },
+
+    asin: function(x) {
+        return new batavia.types.Float(Math.asin(x.__float__().val));
+    },
+
+    // TODO: this isn't supported in PhantomJS
```

Comment:
Looks there is one at https://github.com/MaxArt2501/es6-math -- I will add that.